docs/source/advanced/plugin_dev.rst

```diff
@@ -49,7 +49,7 @@ Plugin types
 
 There are several plugin types:
    * `Plugin`: The general type of plugin. Should contain at least `depends_on = <datakind>`, `provides = <datatype>`, `def compute(self, <datakind>)`, and `dtype = <dtype> ` or `def infer_dtype(): <>`.
-   * `OverlapWindowPlugin`: Allows a plugin to only
+   * `OverlapWindowPlugin`: Allows a plugin to look for data in adjacent chunks. A OverlapWindowPlugin assumes: all inputs are sorted by *endtime*. Since everything This only works for disjoint intervals such as peaks or events, but NOT records! The user has to define get_window_size(self) along with the plugin which returns the required chunk extension in nanoseconds. 
```

Comment:
Think you changed the sentence half way:
`   Since everything This only works `

Reply:
This happens if one is too lazy and just copy pastes sentences from the doc-strings :D I removed the half sentence.

Reply:
Copy pasta strikes again 😉